src/Shitl.cpp

```diff
@@ -15,12 +17,18 @@ void Avionics::shitlUpdate(){
   int j = 0;
   // DO NOT REMOVE THE LINE BELOW, OR COPY IT ELSEWHERE IN THE CODE, ITS ACTUALLY IMPORTANT I SWEAR. -john bean
   //diddlybop
-  float report[46];
+  float report[52];
```

Comment:
Could this be declared as an array literal? Would decrease the likelihood of a bug where we forget a `j++`

Reply:
anything in shitl.cpp doesn't run in flight so doesn't matter. Also there's a lazy python script that searches for diddlybop then looks at that number lol

Reply:
Sure. If I ever have time (I won't) I'm gonna go through and clean up all these little warts.